### PR TITLE
faq: Add outstanding questions

### LINKED ISSUE #69: Add more FAQ questions
## Why do I have to enter my password everytime?

### PR DESCRIPTION
**Pull request by xla**
Closes #69

### CODE REVIEW

File: docs/understanding-radicle/faq.md [hunk 1]
```diff
@@ -125,6 +125,12 @@ access or publish data to the Radicle network - so make sure you keep it safe!
 ## Can I change my passphrase?
 Not yet — so make sure to keep it in a safe place!
 
+## Why do I have to enter my passphrase everytime?
+Interactions through Git and the remote helper are adhoc and don't have the
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:129]:
```suggestion
Interactions through Git and the remote helper are ad-hoc and don't have the
```

File: docs/understanding-radicle/faq.md [hunk 2]
```diff
@@ -125,6 +125,12 @@ access or publish data to the Radicle network - so make sure you keep it safe!
...
+benefit of a long-running daemon - i.e. Upstream client. That means for now the
+passphrase as to be provided every time you interact with Radicle outside of
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:131]:
```suggestion
passphrase has to be provided every time you interact with Radicle outside of
```

File: docs/understanding-radicle/faq.md [hunk 3]
```diff
@@ -140,6 +146,12 @@ found, try refreshing the app. Wait one minute before restarting the app again.
 If you are still running into problems, please submit a request in our [#support
 channel][sc].
 
+## Why am I only connected to one peer?
+By default the Upstream client is conecting to a seed node operated by Radicle.
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:150]:
```suggestion
By default, the Upstream client is conecting to a seed node operated by Radicle.
```

File: docs/understanding-radicle/faq.md [hunk 4]
```diff
@@ -140,6 +146,12 @@ found, try refreshing the app. Wait one minute before restarting the app again.
...
+While we support [epidemic broadcast][eb] to find and connect to other peers, we
+have support [hole punching][hp] just yet, which will prevent a stable conenction
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:152]:
```suggestion
don't have support [hole punching][hp] just yet, which will prevent a stable conenction
```

File: docs/understanding-radicle/faq.md [hunk 5]
```diff
@@ -159,6 +171,13 @@ To join our Matrix chat, follow these steps:
 To join our Matrix chat with an account from another Matrix server, you can use
 this [direct invite link][mc] to join #general.
 
+## Can I run Radicle as a daemon?
+While technically possible we haven't bundled it yet in a convenient package
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:175]:
```suggestion
While this is technically possible, we haven't bundled it yet in a convenient package
```

File: docs/understanding-radicle/faq.md [hunk 6]
```diff
@@ -159,6 +171,13 @@ To join our Matrix chat, follow these steps:
...
+for anyone to run in the background. We are working hard to change that and
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:176]:
```suggestion
for anyone to run in the background. We are working hard to change that so we can
```

File: docs/understanding-radicle/faq.md [hunk 7]
```diff
@@ -159,6 +171,13 @@ To join our Matrix chat, follow these steps:
...
+help people operate Radicle peers in many different ways. Should you be keen to
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:177]:
```suggestion
help people operate Radicle nodes in many different ways. Should you be keen to
```

File: docs/understanding-radicle/faq.md [hunk 8]
```diff
@@ -159,6 +171,13 @@ To join our Matrix chat, follow these steps:
...
+have it as a daemon right, check out how the [seed][si[ is implemented and try to run
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/faq.md:178]: Is this what you wanted this section to be?
```suggestion
have it as a daemon right now, check out how the [seed][si] is implemented, and try to run one on
```

**Review by abbey-titcomb — CHANGES_REQUESTED** [review]
Couple edits

File: docs/understanding-radicle/faq.md [hunk 9]
```diff
@@ -159,6 +171,13 @@ To join our Matrix chat, follow these steps:
...
+your own.
```
Comment (xla) [inline docs/understanding-radicle/faq.md:179]:
```suggestion
have it as a daemon right now, check out how the [seed][si[ is implemented, and
try to run your own.
```

**Pull request merged**